mlflow/utils/autologging_utils/__init__.py

```diff
@@ -223,7 +223,7 @@ def _timed_log_batch(self):
             for i in range(0, len(self.data), MAX_METRICS_PER_BATCH)
         ]
         for metrics_slice in metrics_slices:
-            try_mlflow_log(MlflowClient().log_batch, run_id=current_run_id, metrics=metrics_slice)
+            MlflowClient(self.tracking_uri).log_batch(run_id=current_run_id, metrics=metrics_slice)
```

Comment:
It seems we still need `try_mlflow_log` here:

https://github.com/mlflow/mlflow/pull/4717/checks?check_run_id=3397583060#step:6:1562

```python
____________ test_batch_metrics_logger_continues_if_log_batch_fails ____________

start_run = None

    @pytest.mark.usefixtures(test_mode_off.__name__)
    def test_batch_metrics_logger_continues_if_log_batch_fails(start_run,):
        with mock.patch.object(MlflowClient, "log_batch") as log_batch_mock:
            log_batch_mock.side_effect = [Exception("asdf"), None]
    
            run_id = mlflow.active_run().info.run_id
            with batch_metrics_logger(run_id) as metrics_logger:
                # this call should fail to record since log_batch raised exception
>               metrics_logger.record_metrics({"x": 1}, step=0)
...
E                   Exception: asdf
```